src/App.tsx

```diff
@@ -1,15 +1,28 @@
-import logo from "./logo.svg";
+// import logo from "./logo.svg";
 import "./App.css";
-import Nav from "./Nav/Nav";
+import { Nav } from "./components/Nav/Nav";
+// import { render } from "react-dom";
```

Comment:
One last fix.. can we just delete the code that is commented out? I don't like leaving comments / commented code in the code base. Nice job!